GitHubTraining/Program.cs

```diff
@@ -31,6 +31,7 @@ static void Main(string[] args)
             cars.Add(new Car("Nissan", 2010, "Red"));
             cars.Add(new Car("Bentley", 1950, "Red"));
             cars.Add(new Car("Toyota", 2012, "White"));
+            cars.Add(new Car("Bugati", 1980, "Metalic"));
```

Comment:
Great, thanks! Feel free to complete the request.

Reply:
Great!